### PR TITLE
Storage minor issues

### PR DESCRIPTION
**Pull request by iorveth**
Closes https://github.com/Joystream/joystream/issues/2175

Depends on https://github.com/Joystream/joystream/pull/2173

### CODE REVIEW

File: runtime-modules/storage/src/data_directory.rs [hunk 1]
```diff
@@ -40,9 +39,10 @@ use crate::data_object_type_registry;
 use crate::data_object_type_registry::IsActiveDataObjectType;
 use crate::*;
 
-pub const DEFAULT_QUOTA_SIZE_LIMIT_UPPER_BOUND: u64 = 20000;
-pub const DEFAULT_QUOTA_OBJECTS_LIMIT_UPPER_BOUND: u64 = 200;
-pub const DEFAULT_GLOBAL_QUOTA: Quota = Quota::new(2000000, 2000);
+pub const DEFAULT_VOUCHER_SIZE_LIMIT_UPPER_BOUND: u64 = 20000;
+pub const DEFAULT_VOUCHER_OBJECTS_LIMIT_UPPER_BOUND: u64 = 200;
+pub const DEFAULT_GLOBAL_VOUCHER: Voucher = Voucher::new(2000000, 2000);
+pub const DEFAULT_VOUCHER: Voucher = Voucher::new(50000, 100);
```
Comment (mnaamani) [inline runtime-modules/storage/src/data_directory.rs:45]: I think these default values are very low. The current testnet storage usage is close to 50GB

**Review by mnaamani — APPROVED** [review]
Looks good, we can select more appropriate initial default values and upper bounds for quota, and vouchers. https://github.com/Joystream/joystream/issues/2181

**Review by mnaamani — CHANGES_REQUESTED** [review]
There is actually a clippy error that needs to be resolved.

```
    Checking pallet-storage v3.0.0 (/joystream/runtime-modules/storage)
error: very complex type used. Consider factoring parts into `type` definitions
   --> runtime-modules/storage/src/data_directory.rs:504:19
    |
504 |           vouchers: Vec<(
    |  ___________________^
505 | |             StorageObjectOwner<MemberId<T>, ChannelId<T>, DAOId<T>>,
506 | |             Voucher,
507 | |         )>,
    | |__________^
    |
    = note: `-D clippy::type-complexity` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#type_complexity
```

**Comment by iorveth** [comment]
> Looks good, we can select more appropriate initial default values and upper bounds for quota, and vouchers. https://github.com/Joystream/joystream/issues/2181

Great, thank you for creating an issue for me!

File: runtime-modules/storage/src/data_directory.rs (hunk 1 above)
Comment (iorveth) [reply runtime-modules/storage/src/data_directory.rs:45]: Good point, let`s keep that in mind!

**Comment by iorveth** [comment]
> There is actually a clippy error that needs to be resolved.
> 
> ```
>     Checking pallet-storage v3.0.0 (/joystream/runtime-modules/storage)
> error: very complex type used. Consider factoring parts into `type` definitions
>    --> runtime-modules/storage/src/data_directory.rs:504:19
>     |
> 504 |           vouchers: Vec<(
>     |  ___________________^
> 505 | |             StorageObjectOwner<MemberId<T>, ChannelId<T>, DAOId<T>>,
> 506 | |             Voucher,
> 507 | |         )>,
>     | |__________^
>     |
>     = note: `-D clippy::type-complexity` implied by `-D warnings`
>     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#type_complexity
> ```

Done https://github.com/Joystream/joystream/pull/2176/commits/305b6b092c01773801ca118a774da41e387d99de

**Review by mnaamani — APPROVED** [review]

**Pull request merged**